pages/_app.tsx

```diff
@@ -1,19 +1,25 @@
 import { Box, GlobalStyles } from '@bigcommerce/big-design';
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
 import type { AppProps } from 'next/app';
+import { ThemeProvider } from 'styled-components';
+
 import Header from '../components/header';
 import SessionProvider from '../context/session';
 
 const MyApp = ({ Component, pageProps }: AppProps) => {
     return (
-        <>
+        <ThemeProvider theme={defaultTheme}>
```

Comment:
Adding the `ThemeProvider` builds off of #70 however, the commit is not on main yet.